build-package/setup.py

```diff
@@ -23,14 +23,18 @@
       packages=find_packages(),
       include_package_data=True,
       install_requires=[
+        # NOTE(LA): bumped pandas and changed the scikit-learn version to match the range used by Certifai (common package)
         'numpy>=1.19.5, <1.23',
-        'scikit-learn==0.24.2',
-        'pandas>=0.25.3,<1.2',
+        'scikit-learn>=0.20.3,<1.1',
+        'pandas>=0.23.4,<1.5',
```

Comment:
* Bumped `scikit-learn` to what is used in the certifai common package. The main thing, is that `scikit-learn==1.0.2` has wheels for Apple arm64 on [PyPI](https://pypi.org/project/scikit-learn/1.0.2/#files)
* Similarly matched `pandas` what is used in the certifai common package. I was able to install 'pandas==1.3.0', but that required the C-compiler to build the dependencies and was kinda slow (arm64 wheels are only available for python 3.10). Starting with 'pandas==1.4.0', arm64 wheels are published on [PyPi](https://pypi.org/project/pandas/1.4.0/#files) for Python 3.8.